src/Microsoft.OpenApi.OData.Reader/OpenApiConvertSettings.cs

```diff
@@ -106,9 +106,9 @@ public class OpenApiConvertSettings
         public bool EnableCount { get; set; }
 
         /// <summary>
-        /// Gets/sets a value indicating whether or not to reference @odata.nextLink and @odata.count in responses
+        /// Gets/sets a value indicating whether or not to reference @odata.nextLink, @odata.deltaLink and @odata.count in responses
         /// </summary>
-        public bool RefBaseCollectionPaginationCountResponse { get; set; } = true;
+        public bool EnableODataAnnotationReferencesForResponses { get; set; } = true;
```

Comment:
while this name is more intelligible, it'd be a breaking change with the last minor (stable) we shipped. Revert this change please.

Reply:
I renamed this to also cater for `@odata.deltaLink` since the name was very specific to only `@odata.nextLink` and `@odata.count`. I want to think that the challenge here is we want to have backward compatibility with the version we are targeting. I suggest that we have both settings for now to do the exact same thing and mark ` RefBaseCollectionPaginationCountResponse` as deprecated. The setting will then evaluate to `false` if either one is `false`. 

Thoughts?

Reply:
to paraphrase, `RefBaseCollectionPaginationCountResponse` would be marked as obsolete and be a set of accessors to `EnableODataAnnotationReferencesForResponses`? This way people know the migration path and we can pull the obsolete property on the next major.

Reply:
perfect, go for it then.